### PR TITLE
Add metafile to ConditionalDETR DABDETR and DINO

### PR DESCRIPTION
**Pull request by LYMDLUT**
We (me, @jshilong, @Li-Qingyun, and @KeiChiTse) have refactored DETR-like models, to enhance the usability and readability of our codebase.

This PR  add metafile to support conveniently test.

### CODE REVIEW

File: configs/conditional_detr/metafile.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      URL: https://arxiv.org/abs/2108.06152
+      Title: 'Conditional DETR for Fast Training Convergence'
+    README: configs/conditional_detr/README.md
+    Code:
+      URL: https://github.com/open-mmlab/mmdetection/blob/f4112c9e5611468ffbd57cfba548fd1289264b52/mmdet/models/detectors/conditional_detr.py#L14
+      Version: v3.0.0rc6
+
+Models:
+  - Name: conditional-detr_r50_8xb2-50e_coco.py
+    In Collection: ConditionalDETR
```
Comment (Li-Qingyun) [inline configs/conditional_detr/metafile.yml:23]:
```suggestion
    In Collection: Conditional DETR
```

File: configs/dab_detr/metafile.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Collections:
+  - Name: DABDETR
```
Comment (Li-Qingyun) [inline configs/dab_detr/metafile.yml:2]:
```suggestion
  - Name: DAB-DETR
```

File: configs/dab_detr/metafile.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      URL: https://arxiv.org/abs/2201.12329
+      Title: 'DAB-DETR: Dynamic Anchor Boxes are Better Queries for DETR'
+    README: configs/dab_detr/README.md
+    Code:
+      URL: https://github.com/open-mmlab/mmdetection/blob/f4112c9e5611468ffbd57cfba548fd1289264b52/mmdet/models/detectors/dab_detr.py#L15
+      Version: v3.0.0rc6
+
+Models:
+  - Name: dab-detr_r50_8xb2-50e_coco.py
+    In Collection: DABDETR
```
Comment (Li-Qingyun) [inline configs/dab_detr/metafile.yml:23]:
```suggestion
    In Collection: DAB-DETR
```

**Review by Li-Qingyun — APPROVED** [review]

**Review by Li-Qingyun — APPROVED** [review]

**Comment by RangeKing** [comment]
@LYMDLUT , please fix the lint issue by pre-commit.

File: configs/conditional_detr/metafile.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+Collections:
+  - Name: Conditional DETR
+    Metadata:
+      Training Data: COCO
+      Training Techniques:
+        - AdamW
+        - Multi Scale Train
+        - Gradient Clip
+      Training Resources: 8x a100 GPUs
```
Comment (RangiLyu) [inline configs/conditional_detr/metafile.yml:9]:
```suggestion
      Training Resources: 8x A100 GPUs
```

File: configs/dino/metafile.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
+Collections:
+  - Name: DINO
+    Metadata:
+      Training Data: COCO
+      Training Techniques:
+        - AdamW
+        - Multi Scale Train
+        - Gradient Clip
+      Training Resources: 8x a100 GPUs
```
Comment (RangiLyu) [inline configs/dino/metafile.yml:9]:
```suggestion
      Training Resources: 8x A100 GPUs
```

File: configs/dab_detr/metafile.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+Collections:
+  - Name: DAB-DETR
+    Metadata:
+      Training Data: COCO
+      Training Techniques:
+        - AdamW
+        - Multi Scale Train
+        - Gradient Clip
+      Training Resources: 8x a100 GPUs
```
Comment (RangiLyu) [inline configs/dab_detr/metafile.yml:9]:
```suggestion
      Training Resources: 8x A100 GPUs
```

**Review by RangiLyu — APPROVED** [review]

**Pull request merged**